DataCardProducer/cardConfig7Jet.py

```diff
@@ -0,0 +1,305 @@
+import sys
```

Comment:
I'm trying to understand why there are three versions of this file that look very, very similar. Cannot it not be done via a single file